pkg/curatedpackages/bundle.go

```diff
@@ -0,0 +1,117 @@
+package curatedpackages
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/go-logr/logr"
+
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+	"github.com/aws/eks-anywhere-packages/pkg/artifacts"
+	"github.com/aws/eks-anywhere-packages/pkg/bundle"
+	"github.com/aws/eks-anywhere-packages/pkg/testutil"
+	"github.com/aws/eks-anywhere/pkg/constants"
+	"github.com/aws/eks-anywhere/pkg/executables"
+)
+
+const (
+	Cluster         = "cluster"
+	Registry        = "registry"
+	RegistryBaseRef = "public.ecr.aws/q0f6t3x4/eksa-package-bundles"
+)
+
+func GetLatestBundle(ctx context.Context, kubeConfig string, source string, kubeVersion string) (*api.PackageBundle, error) {
+	var (
+		packageBundle *api.PackageBundle
+		err           error
+	)
+
+	switch strings.ToLower(source) {
+	case Cluster:
+		packageBundle, err = getActiveBundleFromCluster(ctx, kubeConfig)
+	case Registry:
+		packageBundle, err = getLatestBundleFromRegistry(ctx, kubeVersion)
+	}
+
+	if err != nil {
+		return nil, err
+	}
+	return packageBundle, nil
+}
```

Comment:
```suggestion
func GetLatestBundle(ctx context.Context, kubeConfig string, source string, kubeVersion string) (*api.PackageBundle, error) {
	switch strings.ToLower(source) {
	case Cluster:
		return getActiveBundleFromCluster(ctx, kubeConfig)
	case Registry:
		return getLatestBundleFromRegistry(ctx, kubeVersion)
	default:
		return nil, errors.New("unknown source")
	}
}
```

Try not to over complicate things.